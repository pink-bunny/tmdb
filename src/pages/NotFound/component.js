import React from 'react';
import PropTypes from 'prop-types';

const NotFound = ({ location }) => {
  const { pathname } = location;
  return (
    <div className="container">
      <h2>
        No match for
        <code>{pathname}</code>
      </h2>
    </div>
  );
};

NotFound.propTypes = {
  location: PropTypes.shape({
    pathname: PropTypes.string.isRequired
  }).isRequired
};

export default NotFound;
