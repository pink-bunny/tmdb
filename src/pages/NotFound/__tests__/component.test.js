import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../component';

describe('NotFound component', () => {
  it('matches snapshot', () => {
    const location = {
      pathname: '/random'
    };
    const wrapper = shallow(<NotFound location={location} />); /* eslint-disable-line */

    expect(wrapper).toMatchSnapshot();
  });
});
