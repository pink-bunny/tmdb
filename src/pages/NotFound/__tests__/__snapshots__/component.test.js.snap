// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component matches snapshot 1`] = `
<div
  className="container"
>
  <h2>
    No match for
    <code>
      /random
    </code>
  </h2>
</div>
`;
